cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,49 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import HomePageObject from '../support/pages/home.pageObject';
+import { faker } from '@faker-js/faker';
+
+const homePage = new HomePageObject();
+let mainUser, followUser;
+
 describe('User', () => {
   before(() => {
+    cy.task('db:clear');
+
+    mainUser = {
+      email: faker.internet.email(),
+      username: faker.internet.userName(),
+      password: faker.internet.password(10)
+    };
+
+    followUser = {
+      email: faker.internet.email(),
+      username: faker.internet.userName(),
+      password: faker.internet.password(10)
+    };
+
+    cy.register(mainUser.email, mainUser.username, mainUser.password);
+    cy.register(followUser.email, followUser.username, followUser.password);
+  });
 
+  beforeEach(() => {
+    cy.login(mainUser.email, mainUser.username, mainUser.password);
   });
 
-  it.skip('should be able to follow the another user', () => {
+  it('should be able to follow another user', () => {
+    homePage.visitUserProfile(followUser.username);
+    homePage.clickOnFollowBtn();
+
+    homePage.assertBtnName('Unfollow');
```

Comment:
Ensure that the `assertBtnName` method correctly verifies that the button name changes to 'Unfollow' after clicking the follow button. This will confirm that the follow action was successful.